public/app/plugins/panel/canvas/ConnectionAnchors.tsx

```diff
@@ -0,0 +1,259 @@
+import { css } from '@emotion/css';
+import React, { useRef } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+type Props = {
+  setRef: (anchorElement: HTMLDivElement) => void;
+};
+
+export const CONNECTION_ANCHOR_DIV_ID = 'connectionControl';
+
+export const ConnectionAnchors = ({ setRef }: Props) => {
+  const highlightEllipseRef = useRef<HTMLDivElement>(null);
+  const styles = useStyles2(getStyles);
+  const halfSize = 2.5;
+  const halfSizeHighlightEllipse = 5.5;
+  const anchorImage =
+    'data:image/svg+xml;base64,PCFET0NUWVBFIHN2ZyBQVUJMSUMgIi0vL1czQy8vRFREIFNWRyAxLjEvL0VOIiAiaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkIj48c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSI1cHgiIGhlaWdodD0iNXB4IiB2ZXJzaW9uPSIxLjEiPjxwYXRoIGQ9Im0gMCAwIEwgNSA1IE0gMCA1IEwgNSAwIiBzdHJva2Utd2lkdGg9IjIiIHN0eWxlPSJzdHJva2Utb3BhY2l0eTowLjQiIHN0cm9rZT0iI2ZmZmZmZiIvPjxwYXRoIGQ9Im0gMCAwIEwgNSA1IE0gMCA1IEwgNSAwIiBzdHJva2U9IiMyOWI2ZjIiLz48L3N2Zz4=';
+
+  const onMouseEnterAnchor = (event: React.MouseEvent) => {
+    const target = event.target as HTMLImageElement;
+    if (highlightEllipseRef.current && target.style) {
+      highlightEllipseRef.current.style.display = 'block';
+      highlightEllipseRef.current.style.top = `calc(${target.style.top} - ${halfSizeHighlightEllipse}px)`;
+      highlightEllipseRef.current.style.left = `calc(${target.style.left} - ${halfSizeHighlightEllipse}px)`;
+    }
+  };
+
+  const onMouseEnterHighlightElement = () => {
+    // TODO: Implement drawing a connection here (or in scene selecto implementation)
+  };
+
+  const onMouseLeaveHighlightElement = () => {
+    if (highlightEllipseRef.current) {
+      highlightEllipseRef.current.style.display = 'none';
+    }
+  };
+
+  const connectionAnchorAlt = 'connection anchor';
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        display: 'none',
+      }}
+      ref={setRef}
+    >
+      <div
+        id={CONNECTION_ANCHOR_DIV_ID}
+        ref={highlightEllipseRef}
+        className={styles.highlightElement}
+        onMouseEnter={onMouseEnterHighlightElement}
+        onMouseLeave={onMouseLeaveHighlightElement}
+      />
+      <img
```

Comment:
rather than enumerating these explicitly... should it be based off a list of coordinates?   The default set would be:
```
-1,-1
-.5,-1
0,-1
.5,1
1,1
...
``

Reply:
Addressed in [this commit](https://github.com/grafana/grafana/pull/57561/commits/ef3a3ad0a071984b7fee1cc9b269699a1b3b70b1) :) pretty happy with reduction of boiler plate / extensibility for providing custom anchor points

https://user-images.githubusercontent.com/22381771/209012934-df2c3ec2-8c9e-4484-a61a-588bea88000b.mov